lib/prompt/parent_branches.go

```diff
@@ -33,5 +46,82 @@ func askForBranchAncestry(branchName string) {
 }
 
 func askForParentBranch(branchName string) string {
-	panic("unimplemented")
+	branchNameFmt := color.New(color.Bold).Add(color.FgCyan)
+	mainBranch := config.GetMainBranch()
+	message := fmt.Sprintf("Please specify the parent branch of %s by name or number (default: %s): ", branchNameFmt.Sprintf(branchName), mainBranch)
+
+	for {
+		fmt.Printf(message)
```

Comment:
This line should actually be use `fmt.Print` instead of `fmt.Printf`(message)`. I did this as a very minor optimization so that the format only gets compiled once in the case a user enters something invalid. I'll inline it so we have one less variable.